stdlib/experimental/http/requests/requests.flux

```diff
@@ -211,3 +268,30 @@ get = (
         body: body,
         config: config,
     )
+
+// peek converts an HTTP response into a table for easy inspection.
+// The output table includes a **body** column with the response
+// body as a string and a **statusCode** column with the returned
+// status code as an integer.
+//
+// To customize how the response data is structured in a table, use `array.from()`
+// with a function like `json.parse()`. Parse the response body into a set of values
+// and then use `array.from()` to construct a table from those values.
+//
+// ## Parameters
+//
+// - response: Response data from an HTTP request.
+//
+// ## Examples
+//
+// ### Inspect the response of an HTTP request
+//
+// ```
+// import "experimental/http/requests"
+//
+// requests.peek(response: requests.get(
+//     url: "https://api.agify.io",
+//     params: ["name": ["natalie"]],
+// > ))
+// ```
+peek = (response) => array.from(rows: [{statusCode: response.statusCode, body: string(v: response.body)}])
```

Comment:
Wait, why is adding a new property to a record a breaking change? Won't that just be an additive change? If they don't use the field then its not an issue?

Reply:
Consider

```flux
if x then
    request.peek(response)
else
    array.from(rows: [{statusCode: 123, body: "" }])
```

this would start failing typechecks as the two branches do not return the same type if `peek` added more fields